src/server/internalServer.h

```diff
@@ -75,7 +75,8 @@ class InternalServer {
     std::unique_ptr<Response> handle_catalog(const RequestContext& request);
     std::unique_ptr<Response> handle_catalog_v2(const RequestContext& request);
     std::unique_ptr<Response> handle_catalog_v2_root(const RequestContext& request);
-    std::unique_ptr<Response> handle_catalog_v2_entries(const RequestContext& request);
+    std::unique_ptr<Response> handle_catalog_v2_entries(const RequestContext& request, bool partial = false);
```

Comment:
I've put my last comment about the default value for partial on the wrong line but I think it should be changed also here.

Reply:
Done